Dockerfile

```diff
@@ -1,10 +1,6 @@
 FROM node:6-onbuild
 MAINTAINER Trent Oswald (@therebelrobot) <trentoswald@therebelrobot.com>
-RUN npm run install:clean --production
-
-EXPOSE 5465
-
-ENV MONGODB mongodb://localhost:27017/brigadehub-docker
+RUN npm install
 
 ENV GITHUB_ID be1b409d62f41a56684c
```

Comment:
Are these values valid? If not it may be better to use values that are more clearly not valid. If they are valid, I'd recommend pulling them out of version control into a `.env` file that can be sourced as well as passed to `docker` via `--env-file`